components/Button/IconButton.tsx

```diff
@@ -0,0 +1,48 @@
+import clsx from 'clsx'
+export interface IIconButtonProps {
+  isDisabled?: boolean;
+  variant?: 'edit' | 'delete';
+}
+
+const IconButtonInner = ({
```

Comment:
i think it's more clear if we have only one component per file, otherwise looks good to me 👍